src/Microsoft.CSharp/tests/AssignmentTests.cs

```diff
@@ -154,8 +160,8 @@ public void PropertyIncompatibleAssigment(object lhs, object rhs, bool overflow,
             }
         }
 
-        [Theory, MemberData(nameof(PropertyBadAssigments))]
-        public void PropertyIncompatibleConstantAssigment(object lhs, object rhs, bool _, bool invalidCast)
+        [Theory, MemberData(nameof(PropertyIncompatibleConstantAssigment_TestData))]
+        public void PropertyIncompatibleConstantAssigment(object lhs, object rhs, bool invalidCast)
```

Comment:
😕 Which issue was reported here?

Ah, unused theory parameter. We typically solve this by assigning the parameter to a discard:

```suggestion
        public void PropertyIncompatibleConstantAssigment(object lhs, object rhs, bool overflow, bool invalidCast)
```

Then on the first line of the method:

```csharp
// Not used by this test
_ = overflow;
```

Reply:
I decided to use the data transform approach as it's the cleanest one according to the xunit analyzer rules but I'm open for suggestions. Do you think the discard one is cleaner?

Reply:
It's entirely up to the project maintainers to decide. We chose discards because they work with both the XUnit rule and the FXCop rule for unused parameters:

https://github.com/dotnet/roslyn/issues/34441#issuecomment-479171267

Reply:
Thanks, I applied your suggestion and will outline it in the meta-issue.